.github/workflows/ci.yml

```diff
@@ -42,9 +42,6 @@ jobs:
             exit 1
           fi
 
-      - name: Run shellcheck
-        run: shellcheck **/*.sh
```

Comment:
As a result of this PR, we no longer have any shell scripts, so this started complaining... I looked into making the running of it conditional upon finding a file, but the resulting bash had to switch to `find` in order to avoid `nullglob`, and pretty soon it looked more complex than it needed to be given we're simply trying to run shellcheck on a file.

Better IMO to just delete for now and then re-add once we actually have a shell script. Alternatively, I'm fine with commenting out. I realize it might get missed when someone re-adds a shell script, but I still think prefer the clarity of avoiding convoluted bash whose only purpose is to protect against non-existent shell scripts.